projects/plugins/jetpack/extensions/blocks/tiled-gallery/constants.js

```diff
@@ -14,13 +19,14 @@ export const LAYOUT_DEFAULT = 'rectangular';
 export const LAYOUT_SQUARE = 'square';
 export const LAYOUT_STYLES = [
 	{
-		isDefault: true,
+		isDefault: Platform.OS === 'web',
 		name: LAYOUT_DEFAULT,
 	},
 	{
 		name: LAYOUT_CIRCLE,
 	},
 	{
+		isDefault: Platform.OS === 'native',
```

Comment:
@guarani, I came across an issue when testing this across web/native. If you create a gallery on native, it's correctly set to the square layout, but this is undone if you open the editor in a web browser:

1. Create a gallery in the app and confirm it's correctly set to the square layout.
2. Save the post and then reopen its editor in a web browser. 
3. See how the gallery automatically converts to the tiled mosaic layout.

Reply:
I think this is related to an issue I'm seeing where the `className` is not set in the block's serialized representation on mobile. The image below shows web 

| Web (includes `className`) | Mobile (missing `className`) |
| - | - |
| <img width="550" alt="Screen Shot 2022-01-12 at 21 39 18" src="https://user-images.githubusercontent.com/1898325/149245318-18b9a040-a747-4c96-b2c7-d54c75678159.png"> | <img width="394" alt="Screen Shot 2022-01-12 at 21 41 04" src="https://user-images.githubusercontent.com/1898325/149245433-4455309e-c545-4b74-a64d-bad47d0adbe2.png"> |